tests/runtime/test_caikit_health_probe.py

```diff
@@ -208,6 +211,32 @@ def test_health_probe(test_config: ProbeTestConfig):
                     ):
                         # Probe should always pass with both possible servers up
                         assert (
-                            caikit_health_probe.health_probe()
+                            caikit_health_probe.readiness_probe()
                             == test_config.should_become_healthy
                         )
+
+
+@pytest.mark.parametrize(
+    ["proc_identifier", "expected"],
+    [(None, True), ("caikit.runt", True), ("foobar", False)],
```

Comment:
A bit confused with the `caikit.runt` example here - with the `in` check here: https://github.com/caikit/caikit/pull/636/files#diff-e8e3913c46ffe6feeadbaa8ce888d10c282f7b2e38e75d656932755be682295cR104 would this then imply this is just checking for running other (`caikit` vs. `caikit.runtime`?) processes in this particular test case?

Reply:
heh, yeah, sorry, this one is gorpy. `caikit.runt` _should_ correctly satisfy `proc_identifier in arg`, and thus pass the liveness test with a custom override that _does_ match. This is all meant to support the case where the actual runtime process is actually running with some other executable (e.g. `wrapper_server` or something). The third test with `"foobar"` is there to make sure that if there _happens_ to be some other `caikit.runtime` running that is _not_ the server being probed, it doesn't get found by accident.